api/src/main/java/com/nicknathanjustin/streamercontracts/users/UserModel.java

```diff
@@ -0,0 +1,30 @@
+package com.nicknathanjustin.streamercontracts.users;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "users")
+@Builder
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@Setter(value = AccessLevel.PACKAGE)
+@Getter
+public class UserModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String twitchUsername;
```

Comment:
would this ever not be a twitchUsername? if not, than username may be a more descriptive variable name. Another way of looking at it would we ever have more than one username associated with a user? If not, username is probably more descriptive.